scripts/xliff_split_merge/test_xliff_split_merge.bats

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bats
+
+# Setup function to prepare test environment
+setup() {
+  echo "setup"
+}
+
+# Cleanup function to remove test artifacts
+teardown() {
+  #rm -rf output_*
```

Comment:
This line needs to be uncommented to remove the test result after the test is finished.